client/modules/IDE/components/Preferences.jsx

```diff
@@ -131,6 +131,7 @@ class Preferences extends React.Component {
                 <InlineSVG src={plusUrl} alt="Increase Font Size" />
                 <h6 className="preference__label">Increase</h6>
               </button>
+              This will also change console font size
```

Comment:
i don't think this explainer text is necessary here! i think it's logical that changing the font size would change the font in the console as well.